storage/store.go

```diff
@@ -2506,6 +2511,8 @@ func (s *Store) processRaft() {
 			s.pendingRaftGroups.Unlock()
 			s.mu.Unlock()
 
+			s.processRaftMu.Lock()
```

Comment:
You mean grab all of the per-Replica mutexes with `store.mu` held and then release them as we call `handleRaftReady`?